tools/install/ExtractResources.bat

```diff
@@ -11,6 +11,18 @@ IF /I "%2"=="x86" set OPT_Platform=x86
 
 set binroot=%cwd%..\..\bin\%OPT_Platform%\%OPT_CONFIGURATION%
 set wwlroot=%cwd%..\..\bin\wwl
+set nugetroot=%cwd%..\NuGet
+set player=%cwd%..\..\bin\DynamoPlayer
+
+echo -------------------------------------------------------------------------------
+echo NuGet restore playlist packages
+echo DynamoPlayer folder: %player%
+echo %nugetroot%\NuGet.exe restore -ConfigFile %cwd%..\..\dynamo-nuget.config -PackagesDirectory %player%
+echo -------------------------------------------------------------------------------
+rd /s/q %player%
+%nugetroot%\NuGet.exe restore -ConfigFile %cwd%..\..\dynamo-nuget.config -PackagesDirectory %player%
+robocopy "%player%\DynamoPlayer-Revit2017.1.0.0\Revit_2017" "%binroot%\Revit_2017" /e
+robocopy "%player%\DynamoPlayer-SampleScripts-Revit.1.0.0\Revit" "%binroot%\samples\en-US\DynamoPlayer" /e
```

Comment:
We need to keep the NuGet package folder name in sync with its version in packages.config. Did you update the package version, to fix the path issue?